scalability_and_performance/optimization/optimizing-networking.adoc

```diff
@@ -6,9 +6,9 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-The xref:../../networking/openshift_sdn/about-openshift-sdn.adoc#about-openshift-sdn[OpenShift SDN] uses OpenvSwitch, virtual extensible LAN (VXLAN) tunnels, OpenFlow rules, and iptables. This network can be tuned by using jumbo frames, network interface controllers (NIC) offloads, multi-queue, and ethtool settings.
+The xref:../../networking/openshift_sdn/about-openshift-sdn.adoc#about-openshift-sdn[OpenShift SDN] uses OpenvSwitch, virtual extensible LAN (VXLAN) tunnels, OpenFlow rules, and iptables. This network can be tuned by using jumbo frames, multi-queue, and ethtool settings.
 
-xref:../../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#about-ovn-kubernetes[OVN-Kubernetes] uses Geneve (Generic Network Virtualization Encapsulation) instead of VXLAN as the tunnel protocol.
+xref:../../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#about-ovn-kubernetes[OVN-Kubernetes] uses Geneve (Generic Network Virtualization Encapsulation) instead of VXLAN as the tunnel protocol. This network can be tuned by using network interface controllers (NIC) offloads.
```

Comment:
Also this is somewhat out of scope to your changes, but just noting that it seems atypical to mention Geneve and then the fully expanded term in parentheses. I guess it's not a typical abbreviation in the first place (one letter per word in the full term), so I'm not sure if our [guidance about abbreviations](https://www.ibm.com/docs/en/ibm-style?topic=grammar-abbreviations#spelled-out-forms) applies here. 

I'll leave it up to your judgement as to whether it applies, but if you think it does, switch the abbreviation to the parentheses like so:
```suggestion
xref:../../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#about-ovn-kubernetes[OVN-Kubernetes] uses Generic Network Virtualization Encapsulation (Geneve) instead of VXLAN as the tunnel protocol. This network can be tuned by using network interface controllers (NIC) offloads.
```